frontend/src/components/common/header/header.tsx

```diff
@@ -23,13 +27,16 @@ const Header: FC = () => {
   return (
     <header className={styles.header}>
       <div className={styles.headerContainer}>
-        <Link to={AppRoute.SIGN_IN}>
+        <Link to={AppRoute.ROOT}>
           <img className={styles.logo} src={logo} alt="logo" />
         </Link>
         <div className={styles.menu}>
           <div className={styles.profile}>
             <span className={styles.profileName}>
-              {user?.email ?? '-'} @ {tenant?.name ?? '-'}
+              {(user as EAMMasterByIdResponseDto)?.email ??
+                (user as EAMWorkerByIdResponseDto)?.name ??
+                '-'}{' '}
```

Comment:
It is quite hard to read. I think its better to move email and name to separate variables